logstash-core/spec/logstash/config/mixin_spec.rb

```diff
@@ -151,4 +151,50 @@
       expect(subject.params).to include("password")
     end
   end
+
+  context "environment variable injection" do
+    let(:plugin_class) do
+      Class.new(LogStash::Filters::Base)  do
+        config_name "one_plugin"
+        config :oneString, :validate => :string
+        config :oneBoolean, :validate => :boolean
+        config :oneNumber, :validate => :number
+        config :oneArray, :validate => :array
+        config :oneHash, :validate => :hash
+      end
+    end
+    
+    ENV["MIXIN_SPEC_ENV_VAR"] = "123"
+
+    subject {
+      plugin_class.new({
+        "oneString" => "${notExistingVar}",
+        "oneBoolean" => "${notExistingVar:true}",
+        "oneNumber" => "${MIXIN_SPEC_ENV_VAR}",
+        "oneArray" => [ "first array value", "$MIXIN_SPEC_ENV_VAR" ],
+        "oneHash" => { "key" => "$MIXIN_SPEC_ENV_VAR" }
+      })
+    }
+     
+    it "should have oneString param as empty string (env var not found)" do
```

Comment:
Can we add spec for multiple env variables in the same config? For example `MIXIN_SPEC_ENV_VAR` `MIXIN_SPEC_ENV_VAR2` .. and make sure they are replaced accordingly.